eXo/Sources/Controllers/HomePage/HomePageViewController.swift

```diff
@@ -275,6 +287,14 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
             print("=============== Navigation Url : \(urlToSee)")
         }
         
+        // Open the Jitsi call.
+        
+        if request.url?.absoluteString.range(of:"?jwt=") != nil {
```

Comment:
Could we check on the Jitsi URL instead of checking just for the jwt token ? we may have integration with JWT for other applications

Reply:
Done